Sources/ScipioKit/Producer/Cache/CacheSystem.swift

```diff
@@ -148,28 +148,37 @@ struct CacheSystem {
     }
 
     init(
-        descriptionPackage: DescriptionPackage,
+        pinsStore: PinsStore,
         outputDirectory: URL,
         storage: (any CacheStorage)?,
         fileSystem: any FileSystem = localFileSystem
     ) {
-        self.descriptionPackage = descriptionPackage
+        self.pinsStore = pinsStore
         self.outputDirectory = outputDirectory
         self.storage = storage
         self.fileSystem = fileSystem
     }
 
+    private struct TaskContext: Sendable {
+        var outputDirectory: URL
+        var frameworkName: String
+    }
+
     func cacheFrameworks(_ targets: Set<CacheTarget>) async {
         let chunked = targets.chunks(ofCount: storage?.paralellNumber ?? CacheSystem.defaultParalellNumber)
 
         for chunk in chunked {
             await withTaskGroup(of: Void.self) { group in
                 for target in chunk {
+                    let context = TaskContext(
+                        outputDirectory: outputDirectory,
+                        frameworkName: target.buildProduct.frameworkName
+                    )
```

Comment:
Is this context required?
`URL` and `CacheTarget` are Sendable, so I think we can directory use them in `addTask`.